frontend/src/store/metadataStore.ts

```diff
@@ -87,10 +87,10 @@ export const useMetadataStore = defineStore('metadata', () => {
     }
   }
 
-  async function searchMetadata(metadataSet: Array<MetadataPair> = []) {
+  async function searchMetadata(bucketId: string = '', metadataSet: Array<MetadataPair> = []) {
```

Comment:
bucketId should be the second argument, and probably should not hard default to empty string when unspecified, as that will limit this store action's utility in the long run in the event you need to do a COMS-wide search.